docs/gettingstarted/installation.rst

```diff
@@ -100,10 +100,10 @@ You can see the Homebrew recipe in the `homebrew-tap`_ repository.
 
 The ``flocker-deploy`` command line program will now be available:
 
-.. code-block:: console
+.. version-code-block:: console
 
    alice@mercury:~$ flocker-deploy --version
-   0.3.2
+   |latest-packaged-version|
```

Comment:
I'm not sure if it occurs in the PR, but this also refers to the version of the node software (they should always be the same, for the moment).